uw-frame-components/portal/storage/services.js

```diff
@@ -36,11 +36,7 @@ define(['angular', 'jquery'], function(angular, $) {
 
       // public functions
       var isKVStoreActivated = function() {
-        if (SERVICE_LOC.kvURL) {
-          return true;
-        } else {
-          return false;
-        }
+        return SERVICE_LOC.kvURL;
```

Comment:
possibly prefix with `!!` to cast into a boolean so you're not just returning the string on an is___ method call